azurerm/helpers/authentication/access_token_test.go

```diff
@@ -163,7 +163,7 @@ func TestAzureFindValidAccessTokenForTenant_ValidFromCloudShell(t *testing.T) {
 		t.Fatalf("Expected the Client ID to be %q but got %q", expectedToken.ClientID, token.ClientID)
 	}
 
-	if token.IsCloudShell != true {
+	if token.IsCloudShell {
```

Comment:
I think this inverted the boolean?

Reply:
It sure did

Reply:
🤔 I'd like to hold off on this PR until #2199 is merged; since otherwise there's going to be a ton of merge conflicts which'll be hard to resolve - WDYT?

Reply:
👍

Reply:
@katbyte I think this should be safe to `git reset HEAD~1 && [lint]` now?

Reply:
@tombuildsstuff, 

updated and ready to go